src/dir_status.ml

```diff
@@ -111,8 +116,9 @@ module DB = struct
         | Some No ->
           Standalone (Some (ft_dir, Some d))
         | None ->
-          if dir <> project_root
-          then begin
+          if build_dir_is_project_root then
```

Comment:
The `<>` is now `=` because I flipped the order of the branches. The shorter branch comes first this way.